### DISCUSSION TITLE
Help fixing black docs build with furo 2023.8.19

### DISCUSSION THREAD

**Discussion by cooperlees** [pradyunsg/furo#712] [Questions] [answered]
Howdy all,

We seem to be running into the following error:
```
Configuration error:
This documentation is not using `furo.css` as the stylesheet. If you have set `html_style` in your conf.py file, remove it.
```
- Actions fails I can repro locally: https://github.com/psf/black/actions/runs/5922834942/job/16057411149?pr=3848
- I read https://github.com/pradyunsg/furo/discussions/299

I tried commenting out anything I thought could be causing an issues but can not work out our cause as we don't set `html_tyle` from what I can see in the [black/docs/conf.py](https://github.com/psf/black/tree/main/docs/conf.py). Any thoughts on what's causing this?

**Comment by dgw** [comment]
See #693, #708, possibly #710

**Comment by cooperlees** [comment, marked as answer]
Thanks. It was upgrade to sphinx 7.2.3 and fix `intersphinx_mapping` in the config.
